test/unit/utils.test.js

```diff
@@ -250,6 +250,27 @@ describe(Support.getTestDialectTeaser('Utils'), () => {
     });
   });
 
+  describe('Interleave', () => {
+    it('interleaves',  () => {
+      expect(Utils.interleave(['a', 'b', 'c'], ',')).to.deep.equal(['a', ',', 'b', ',', 'c']);
+    });
+
+    it('Returns new array', () => {
```

Comment:
Nit: Since these strings are effectively used for building full test names you generally want to keep it lowercased, not capitalised.

Reply:
You are right